plugin/src/main/java/net/dzikoysk/funnyguilds/user/UserManager.java

```diff
@@ -201,6 +205,27 @@ public boolean playedBefore(String nickname, boolean ignoreCase) {
         return findByName(nickname, ignoreCase).isPresent();
     }
 
+
+    public String getUserPosition(User user) {
+        if (user.isOwner()) {
+            return pluginConfiguration.chatPositionLeader;
+        }
+
+        if (user.isDeputy()) {
+            return pluginConfiguration.chatPositionDeputy;
+        }
+
+        return pluginConfiguration.chatPositionMember;
+    }
+
+    public String getUserPosition(Option<User> userOption) {
+        if (userOption.isEmpty()) {
+            return "";
+        }
+
+        return this.getUserPosition(userOption.get());
+    }
```

Comment:
to raczej nie tutaj

Reply:
![image](https://user-images.githubusercontent.com/44551064/165111341-6d577ef0-83da-4512-8065-6266647fd39e.png)